src/pages/UpdateServers.js

```diff
@@ -191,13 +193,22 @@ class UpdateServers extends BaseUpdateWizardPage {
     return (<div className='notification-message-container'>{msgList}</div>);
   }
 
-  handleShowMenuReplaceServer = (server) => {
-    this.setState({
-      showReplaceModal: true,
-      serverToReplace: server
-    });
+  checkSharedControllerDeployer = (server) => {
+    fetchJson('api/v1/ips')
```

Comment:
its a little unclear what the business logic is here.. I assume its getting a list of ips of the current server (which would be the deployer since the UI is hosted there) and then checking it against the IP of the server being replaced ... I guess I'm fuzzy on api/v1/ips...

Reply:
Yes, that's exactly what it does. FWIW, SelectServersToProvision also has similar logic to avoid doing an OS provision of a node if it is the one hosting the deplyer.